docs/source/dvc_steps.rst

```diff
@@ -183,7 +186,10 @@ This parameter has a strong effect on computation time, so be careful.
 You can then either run a **Single** run, or a **Bulk** run:
 
 - A **single run** will run with the current point cloud you have generated, you only need to select the number of sampling points in the subvolume region.
-- If you select to run in **bulk**, this will generate multiple point clouds and perform runs on them, instead of your current point cloud. You can set the minimum and maximum subvolume size you would like, and the size of the step between these values, and similar for the sampling points. In the example above, this would perform runs on point clouds with sizes 30, 40 and 50, and number of sampling points 9000, 9500 and 10000, so 9 runs in total. Note that the other settings for the point clouds generated will be taken from what you selected on the point cloud panel, including the subvolume shape, dimensionality, overlaps and rotation angles.
+- If you select to run in **bulk**, this will use the loaded or generated point cloud and run dvc analysis changing the parameters **subvolume size** and 
+**sampling points in subvolume**. You can set the minimum and maximum subvolume size you would like, and the size of the step between these values, and similar
+ for the sampling points. In the example above, this would perform runs on point clouds with sizes 30, 40 and 50, and number of sampling points 1000, 2000, 3000,
```

Comment:
```suggestion
 for the sampling points. In the example above, this would perform runs on point clouds with subvolume sizes 30, 60 and 90, and number of sampling points 1000, 2000, 3000,
```

numbers need updating because the screenshot has been updated with different numbers